wire-runtime/api/wire-runtime.api

```diff
@@ -32,22 +32,6 @@ public final class com/squareup/wire/AnyMessage$Companion {
 	public final fun pack (Lcom/squareup/wire/Message;)Lcom/squareup/wire/AnyMessage;
 }
 
-public final class com/squareup/wire/DoubleProtoAdapter : com/squareup/wire/ProtoAdapter {
```

Comment:
Is this like picasso where CI would fail if we don't manually update ourselves those files?

Reply:
pretty sweet https://github.com/Kotlin/binary-compatibility-validator#tasks

Reply:
Exactly!